src/test/user-mutation.test.ts

```diff
@@ -0,0 +1,113 @@
+
+const NAME = "Hugo"
+const EMAIL = "hugo@gmail.com"
+const EMAIL2 = "joao@gmail.com"
+const PASSW = "password1"
+const CPF = "12345678900"
+const BDATE = "04/06/1993"
+
+describe('User mutation tests', () => {
+
+  let agent;
+  let repository: Repository<User>;
+
+  const userMutation = (variable: {data}, token) => {
+
+    const query = `
+    mutation ( $data: CreateUserInput! ){
+      createUser(data: $data) {
+        id 
+        name 
+        email 
+        birthDate 
+        cpf
+
+      }
+    }
+    `
+    return agent.post("/").send({ query: query, variables: variable }).set('Authorization', token)
+  }
+
+  before(async () => {
+    agent = requests('http://localhost:4000');
+    repository = getRepository(User);
+
+    const user = new User();
+
+    user.name = NAME;
+    user.email = EMAIL;
+    user.birthDate = BDATE;
+    user.cpf = CPF;
+    user.password = hashPassword(PASSW);
+
+    await repository.save(user);
+
+  })
+
+  after( async () => {
+    await repository.delete({ email: EMAIL})
+    await repository.delete({ email: EMAIL2})
+  })
+
+  it('Example user created', async function() {
+    expect(NAME).to.be.equals((await getRepository(User).findOne({email: EMAIL})).name);
+    expect(BDATE).to.be.equals((await getRepository(User).findOne({email: EMAIL})).birthDate);
+    expect(CPF).to.be.equals((await getRepository(User).findOne({email: EMAIL})).cpf);
+    expect(hashPassword(PASSW)).to.be.equals((await getRepository(User).findOne({email: EMAIL})).password);
+  })
+
+  it('System should not allow two or more users with the same e-mail', async function() {
+    var token = jwt.sign({ userId: 1 }, APP_SECRET, { expiresIn: TEN_MINUTES }); 
+    const res = await userMutation( {data: { name: NAME, email:EMAIL, birthDate:BDATE, cpf:CPF, password:PASSW }}, token);
+    expect(res.body.errors[0].message).to.be.equals(EMAIL_DUPLICATED);
+  });
+
+  it('Password should have at least 1 digit', async function() {
+    var token = jwt.sign({ userId: 1 }, APP_SECRET, { expiresIn: TEN_MINUTES }); 
+    const res = await userMutation( {data: { name: NAME, email:EMAIL2, birthDate:BDATE, cpf:CPF, password:"password" }}, token);
+    expect(res.body.errors[0].message).to.be.equals(PASSW_DIGIT);
+  })
+
+  it('Password should have at least 1 digit', async function() {
+    var token = jwt.sign({ userId: 1 }, APP_SECRET, { expiresIn: TEN_MINUTES }); 
+    const res = await userMutation( {data: { name: NAME, email:EMAIL2, birthDate:BDATE, cpf:CPF, password:"1234567890" }}, token);
+    expect(res.body.errors[0].message).to.be.equals(PASSW_LETTERS);
+  })
+
+  it('Password should have at least 1 digit', async function() {
+    var token = jwt.sign({ userId: 1 }, APP_SECRET, { expiresIn: TEN_MINUTES }); 
+    const res = await userMutation( {data: { name: NAME, email:EMAIL2, birthDate:BDATE, cpf:CPF, password:"pass1" }}, token);
+    expect(res.body.errors[0].message).to.be.equals(PASSW_SHORT);
+  })
```

Comment:
Os 3 testes tem o mesmo enunciado. Eu sugiro você começar todos com "should" tbm.

`should check if password has at least one digit`